### DISCUSSION TITLE
is it possible to put links in footer copyright

### DISCUSSION THREAD

**Discussion by peguine** [facebook/docusaurus#6628] [Q&A] [answered]
```
<a href='https://xxx.com/' target="_blank">
       copyright @2022 All Rights Reserved
</a>
```

**Comment by slorber** [comment, marked as answer]
Yes, you can use html strings in the copyright config, as React will render this HTML

```jsx

{footer.copyright ? (
              <div
                className="footer__copyright"
                dangerouslySetInnerHTML={{
                  __html: footer.copyright,
                }}
              />
            ) : null}
```

For more advanced use-cases, you can swizzle the Footer component

**Reply by peguine** [reply]
copyright: `<div>abc © ${new Date().getFullYear()}  All Rights Reserved.<a href="https://xxx.com/">footer</a></div>`,

works,thanks

**Reply by iqfareez** [reply]
I got an error 
```
[ERROR] D:\Iqmal\Documents\webdev stuff\iiumscwebsite\docusaurus.config.js:118
        copyright: <div>abc  All Rights Reserved.<a href="https://xxx.com/">footer</a></div>,
                   ^

SyntaxError: Unexpected token '<'

```

How did you make it works? I tried to surround with double quotes, but no success.

Edit: nvm, I got it to works now

`copyright: '<div>Copyright © 2022 <a href="https://iqfareez.com">Muhammad Fareez<a><div>',`

**Reply by slorber** [reply]
@iqfareez your a and div closing tags are not good, you should rather to use `</a></div>`

**Reply by iqfareez** [reply]
Good catch! Thanks man. Wonder how can I miss that one. 🤷
